spoon-runner/src/main/java/com/squareup/spoon/SpoonDeviceRunner.java

```diff
@@ -196,6 +198,17 @@ public DeviceResult run(AndroidDebugBridge adb) {
       logDebug(debug, "About to actually run tests for [%s]", serial);
       RemoteAndroidTestRunner runner = new RemoteAndroidTestRunner(testPackage, testRunner, device);
       runner.setMaxtimeToOutputResponse(adbTimeout);
+
+      if(instrumentationArgs != null && instrumentationArgs.size() > 0){
```

Comment:
on this and the next several lines, add a space before the `(` and after the `)`